iron-list.html

```diff
@@ -1170,7 +1271,7 @@
 
       // scroll to the item as much as we can
       while (currentVirtualItem < idx && targetOffsetTop < hiddenContentSize) {
-        targetOffsetTop = targetOffsetTop + this._physicalSizes[currentTopItem];
+        targetOffsetTop = targetOffsetTop + this._getPhysicalTopIncrement(currentTopItem);
         currentTopItem = (currentTopItem + 1) % this._physicalCount;
         currentVirtualItem++;

```

Comment:
I found a bug in `_getPhysicalTopIncrement` that I believe I have resolved. In my initial testing resize seemed to be functioning much better. Are you still seeing the same issues?